Cabal/doc/developing-packages.rst

```diff
@@ -166,23 +213,23 @@ package, so we must list it:
     library
       exposed-modules:     Proglet
       other-modules:
-      build-depends:       containers, base == 4.*
+      build-depends:       containers, base >=4.xx && <4.yy
```

Comment:
Above was valid syntax, below isn't.

Reply:
Could you clarify? I'm not sure what you mean.

Reply:
you can literally write `base ==4.*` in the cabal file, you cannot `>= 4.xx && <4.yy`. I don't like that we have code snippets which aren't "real".

IIRC there was an issue about the fact that we e.g. use `pkg-x.y.z` and sometimes that `pkg` is a variable, and sometimes it is literal. We don't have a solution to that (I'd use slated text for variables, but we have no means to format so), so better to stick to literal text when possible.

It is IMO lesser band if the bounds are different then what the current `cabal-install` executable would generate. I guess it depends on GHC generated, and that difference between text and what user sees ought to be explained.

TL;DR don't make user think which parts are "patterns" and which are literal.

Reply:
Ahh gotcha, that makes sense. Changed to `>=4.11 & <4.12` throughout page.